builtin.c

```diff
@@ -63,6 +63,30 @@ static void S_finish_export_lexical(pTHX)
     LEAVE;
 }
 
+XS(XS_builtin_load);
+XS(XS_builtin_load)
+{
+    dXSARGS;
+
+    if (items != 1) {
+        croak_xs_usage(cv, "arg");
+    }
+
+    SV *module = TOPs;
+
+    if (!SvOK(module)) {
+        Perl_croak(aTHX_ "Undefined argument to builtin::load");
+    }
+
+    module = newSVsv(module);
+
+    if (!SvPOK(module))
+        SvPV_force_nolen(module);
+
+    Perl_load_module(aTHX_ PERL_LOADMOD_NOIMPORT, module, NULL, NULL, NULL);
```

Comment:
I think only one `NULL` value here is necessary (a second would have been if flags hadn't been `PERL_LOADMOD_NOIMPORT`).